parl/remote/remote_constants.py

```diff
@@ -22,4 +22,5 @@
 
 NORMAL_TAG = b'[NORMAL]'
 
+# seconds interval of heartbeat mechanism
```

Comment:
interval of heartbeat mechanism in the unit of second

Reply:
done